inc/saiswitch.h

```diff
@@ -2141,6 +2147,69 @@ typedef enum _sai_switch_attr_t
      */
     SAI_SWITCH_ATTR_QOS_TC_AND_COLOR_TO_MPLS_EXP_MAP,
 
+    /**
+     * @brief Vendor specific switch ID. Identifies switch chip
+     *
+     * Mandatory in VOQ Switch
+     *
+     * @type sai_uint32_t
+     * @flags MANDATORY_ON_CREATE | CREATE_ONLY
+     */
+    SAI_SWITCH_ATTR_SWITCH_ID,
+
+    /**
+     * @brief Maximum number of cores in the VOQ Switch (chassis)
+     *
+     * @type sai_uint32_t
+     * @flags MANDATORY_ON_CREATE | CREATE_ONLY
+     */
+    SAI_SWITCH_ATTR_MAX_SYSTEM_CORES,
+
+    /**
+     * @brief System port configuration list.
+     *
+     * @type sai_system_port_config_list_t
+     * @flags CREATE_ONLY
+     * @default empty
```

Comment:
This attribute applies only if the switch type. The condition should be specified here.
"@condition SAI_SWITCH_ATTR_TYPE == SAI_SWITCH_TYPE_VOQ"

Reply:
Will update the meta condition

Reply:
Agreed. Making the change. Also need to change to MANDATORY_ON_CREATE to satisfy metachecker.